src/azure-cli/service_name.json

```diff
@@ -0,0 +1,688 @@
+    "URL": "https://docs.microsoft.com/azure/azure-app-configuration/"
+  },
+  {
+    "Command": "az appservice",
+    "Description": "Manage App Service plans.",
+    "AzureServiceName": "Azure App Service",
+    "ConversationalServiceName": "App Service",
+    "URL": "https://docs.microsoft.com/azure/app-service/"
+  },
+  {
+    "Command": "az aro",
+    "Description": "Manage Azure Red Hat OpenShift clusters.",
+    "AzureServiceName": "Azure Red Hat OpenShift",
+    "ConversationalServiceName": "Red Hat OpenShift",
+    "URL": "https://docs.microsoft.com/azure/openshift/"
+  },
+  {
+    "Command": "az artifacts",
+    "Description": "Manage Azure Artifacts.",
+    "AzureServiceName": "Azure Artifacts",
+    "ConversationalServiceName": "Artifacts",
+    "URL": "https://docs.microsoft.com/azure/devops/artifacts/"
+  },
+  {
+    "Command": "az backup",
+    "Description": "Manage Azure Backups.",
+    "AzureServiceName": "Azure Backup",
+    "ConversationalServiceName": "Backup",
+    "URL": "https://docs.microsoft.com/azure/backup/"
+  },
+  {
+    "Command": "az batch",
+    "Description": "Manage Azure Batch.",
+    "AzureServiceName": "Azure Batch",
+    "ConversationalServiceName": "Batch",
+    "URL": "https://docs.microsoft.com/azure/batch/"
+  },
+  {
+    "Command": "az batchai",
+    "Description": "Manage Batch AI resources.",
+    "AzureServiceName": "Azure Machine Learning",
+    "ConversationalServiceName": "Machine Learning",
+    "URL": "https://docs.microsoft.com/previous-versions/azure/batch-ai/how-to-migrate"
+  },
+  {
+    "Command": "az billing",
+    "Description": "Manage Azure Billing.",
+    "AzureServiceName": "Azure Cost Management + Billing",
+    "ConversationalServiceName": "Cost Management + Billing",
+    "URL": "https://docs.microsoft.com/azure/cost-management-billing/"
+  },
+  {
+    "Command": "az boards",
+    "Description": "Manage Azure Boards.",
+    "AzureServiceName": "Azure Boards",
+    "ConversationalServiceName": "Boards",
+    "URL": "https://docs.microsoft.com/azure/devops/boards/"
+  },
+  {
+    "Command": "az bot",
+    "Description": "Manage Microsoft Azure Bot Service.",
+    "AzureServiceName": "Azure Bot Service",
+    "ConversationalServiceName": "Bot Service",
+    "URL": "https://docs.microsoft.com/azure/bot-service/"
+  },
+  {
+    "Command": "az cache",
+    "Description": "Commands to manage CLI objects cached using the\u00a0--defer\u00a0argument.",
+    "AzureServiceName": "Azure CLI",
+    "ConversationalServiceName": "Azure CLI",
+    "URL": "https://docs.microsoft.com/cli/azure/"
+  },
+  {
+    "Command": "az cdn",
+    "Description": "Manage Azure Content Delivery Networks (CDNs).",
+    "AzureServiceName": "Azure Network",
+    "ConversationalServiceName": "Network",
+    "URL": "https://docs.microsoft.com/azure/cdn/"
+  },
+  {
+    "Command": "az cloud",
+    "Description": "Manage registered Azure clouds.",
+    "AzureServiceName": "Azure cloud services",
```

Comment:
this is Not auzre cloud service, but should belongs to CLI core part. Pls work with Delora to fix excel.

Reply:
All Azure CLI core stuff are named "Azure CLI" in AzureServiceName now